features/Hair Specular/Shaders/Hair/Hair.hlsli

```diff
@@ -4,12 +4,20 @@
 #include "Common/BRDF.hlsli"
 #include "Common/Math.hlsli"
 
-// #define MARSCHNER
+#define HAIR_LIGHTING_MULTIPLIER Math::PI  // Compensating to adapt to vanilla lighting model
+#define GAME_UNIT_TO_CM 1.428f
```

Comment:
We don't have this defined somewhere in hlsl? I know I added it for cpp.

@coderabbitai can you confirm if we have this defined anywhere for hlsl or otherwise suggest a good place.